dist/tools/randsort/randsort.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import random
+import secrets
+
+def secrets_random():
+    maxval = 2 ** 32
+    return secrets.randbelow(maxval) / maxval
+
+input = sys.stdin.readlines()
+try:
+    os.environ["RIOTINSECURE"]
```

Comment:
The name of this environment variable is currently being discussed in #13175.

Reply:
This needs to be factored out. Better not have those cludges in an otherwise generally useful tool.

Reply:
I didn't want to introduce an additional if/else Branch in the `$(ELFFILE)` Makefile target. See also: https://github.com/RIOT-OS/RIOT/pull/13176#discussion_r368969146